### DISCUSSION TITLE
[Suggestion] Can you please upload the compiled package to a public repository such as maven? Thank you. [13734]

### DISCUSSION THREAD

**Discussion by liberty724048** [eProsima/Fast-DDS-Gen#238] [Ideas]
Sometimes it's difficult to set up the compiling environment of Fast-DDS-Gen. Even if every dependency is installed, the idlparser.jar is still uneasy to compile due to internet blocks

**Comment by JLBuenoLopez** [comment]
Labeling this as an enhancement and moving it to the Ideas discussion forum where enhancements and feature requests are discussed.

**Comment by Ryanf55** [comment]
What about an `apt` package on ubuntu? I would love to be able to:
`apt install fastddsgen`
